src/components/PageNotFound/PageNotFound.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './PageNotFound.scss';
+
+export const PageNotFound: React.FC = () => (
+  <div className="container-page-not-found">
+    <img src="../../../public/img/page-not-found.png" alt="page-not-found" />
```

Comment:
if your files are in a folder `public`
you can write like this `src="/img/page-not-found.png"`